test/integration/roles/test_amazon_rds/tasks/main.yml

```diff
@@ -0,0 +1,155 @@
+# create and kill an RDS
+
+# Improvements expected in later versions of this test case
+
+# use present instead of create
+# use absent instead of delete
+# use SSD
+
+
+# Start with creation - N.B. if the RDS already exists from a previous run this
+# may not change anything so we don't insist on that
+
+  - name: create RDS
+    tags: test_amazon_rds
+    local_action:
+      module: rds
+      region: "{{ ec2_region }}"
+      command: create
+      instance_name: "{{ resource_prefix }}-rds"
+      db_engine: postgres
+      size: 10
+      instance_type: "db.t2.micro"
+      username: "{{ rds_master_db_user }}"
+      password: "{{ rds_db_master_password }}"
+      publicly_accessible: no
+      wait: yes
+      wait_timeout: 1200 # 600 doesn't cut it!!
+    environment:
+      EC2_REGION: '{{ec2_region}}'
+      EC2_ACCESS_KEY: '{{ec2_access_key}}'
```

Comment:
These should be picked up automatically. Don't have an ec2_access_key variable - use the proper environment variables (`AWS_ACCESS_KEY` etc) or, preferably, just use `.aws/credentials` with a profile if necessary